public/app/features/manage-dashboards/components/PublicDashboardListTable/PublicDashboardListTable.test.tsx

```diff
@@ -212,7 +212,7 @@ const renderPublicDashboardItemCorrectly = (pd: ListPublicDashboardResponse, idx
   const rowDataCells = within(tableRow).getAllByRole('cell');
   expect(rowDataCells).toHaveLength(3);
 
-  const statusTag = within(rowDataCells[1]).getByText(pd.isEnabled ? 'enabled' : 'disabled');
+  const statusTag = within(rowDataCells[1]).getByText(pd.isEnabled ? 'enabled' : 'paused');
```

Comment:
For now, it's ok, but I think we should reconsider if we want to continue showing 'enabled'. We can review this when we work on a redesign of the audit page.